packages/app/ui/tamagui.config.ts

```diff
@@ -543,6 +543,7 @@ export const config =
         themes,
         media,
         settings: {
+          defaultFont: 'body',
```

Comment:
Including this, for some reason, ensures that the theme actually uses `body` as the defaultFont. We shouldn't *need* to include it, since the default for this is `body` as-is, but we weren't getting our font variables without it.